app/main/presenters/search_presenters.py

```diff
@@ -208,13 +217,11 @@ def build_lots_and_categories_link_tree(framework, lots, category_filter_group,
     root_node['children'] = list()
     selected_filters.append(root_node)
 
-    # If we're searching against a specific lot, we should only build the tree for that lot.
-    if current_lot_slug:
-        lots = list(filter(lambda lot: lot['slug'] == current_lot_slug, lots))
-
-    aggregations_by_lot = {lot['slug']: _get_aggregations_for_lot_with_filters(lot['slug'], content_manifest,
-                                                                               framework, request)
-                           for lot in lots}
+    aggregations_by_lot = {
+        lot['slug']: _get_aggregations_for_lot_with_filters(
+            lot['slug'], content_manifest, framework, cleaned_request_args, doc_type, index
+        ) for lot in lots
+    }
 
     for lot in lots:
         selected_categories = []
```

Comment:
`lot_selected = (lot['slug'] == current_lot_slug)`
extraneous parenthesis
Again slightly out of scope.

Reply:
I like them as it highlights that it's doing an `==`, not just a three-way `=`

Reply:
Have looked at it both ways and I agree with Rob i think

Reply:
Fair enough.

I think these are good reasons not to:
 * https://google.github.io/styleguide/pyguide.html?showone=Parentheses#Parentheses
 * Extra, extraneous characters to parse
 * Brackets in python are commonly used to indicate the definition of a tuple
 * It breaks convention which reduces confidence

My 2 cents:
So this is a bit more conceptual and a personal preference thing but I feel like it discourages recognition of the action of the `lot['slug'] == current_lot_slug` part. 
This piece of code _will always_ be evaluated. It doesn't need the assignment or the brackets. Because it's actually, effectively, a function call.
In this case it's equivalent to  `str.__eq__(lot['slug'], current_lot_slug)`

If it was a ternary statement it would have an `if` which should be obvious if you've got syntax highlighting

Reply:
You make good points Ben. I'm going to remove them.